apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Alert, Linking, Platform, View } from 'react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { Text } from '@fluentui/react-native';
+import { stackStyle } from '../Common/styles';
+import { IKeyboardEvent, IHandledKeyboardEvent } from '@office-iss/react-native-win32';
+
+export const PressableUsage: React.FunctionComponent<{}> = () => {
+  if (Platform.OS == ('win32' as any)) {
+    const _onPress = (): void => {
+      Linking.openURL('www.microsoft.com');
+    };
+
+    const _onKeyDown = (ev: IKeyboardEvent) => {
+      if (ev.nativeEvent.key === 'Enter' || ev.nativeEvent.key == 'Space') Linking.openURL('www.microsoft.com');
+    };
+
+    const _onPress2 = (): void => {
+      Alert.alert('Alert', 'Success!');
+    };
+
+    const _onKeyDown2 = (ev: IKeyboardEvent) => {
+      if (ev.nativeEvent.key === 'Enter' || ev.nativeEvent.key == ' ') Alert.alert('Alert', 'Success!');
+    };
+
+    const handledNativeKeyboardEvents: IHandledKeyboardEvent[] = [{ key: 'Enter' }, { key: 'Space' }];
+
+    return (
+      <View>
+        <Stack style={stackStyle} gap={5}>
+          <Text variant={'bodyStandard'}>
+            To learn more about microsoft, visit this{' '}
```

Comment:
What does the rendered output look like? Is there a formatting thing I'm missing?

Reply:
Are you referencing the `{' '}`? It's just a trailing space on the first part of the sentence. It looks like a regular sentence with an inline link.

Reply:
yes, nice. Text components don't render trailing whitespace? For a second, I thought there was some crazy template literal type stuff going on

Reply:
I think they do, it's just added in by the editor